patch/bootloader-backdoor-custom.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/source/ti/devices/.meta/templates/ti_devices_CC26XX_config.c.xdt b/source/ti/devices/.meta/templates/ti_devices_CC26XX_config.c.xdt
+index 9e1defe4e..852635663 100644
+--- a/source/ti/devices/.meta/templates/ti_devices_CC26XX_config.c.xdt
++++ b/source/ti/devices/.meta/templates/ti_devices_CC26XX_config.c.xdt
+@@ -155,11 +155,11 @@
+ 
+         % var diohex = inst.dioBootloaderBackdoor.toString(16);
+ // DIO number for boot loader backdoor
+-#define SET_CCFG_BL_CONFIG_BL_PIN_NUMBER                0x`diohex`
```

Comment:
Why put the value instead of patching "lprf_ccfg_settings.js" file ?
![image](https://github.com/user-attachments/assets/31963ca2-bf36-40a7-b773-2c275d0aa42a)

Reply:
Tried to change only this one as it seems to be the main param (overwritting Device param in .syscfg file), but didn't work on its own. Maybe it would work to update this one instead of the template but it will require more test to try and I won't be able to remove the patch yet as there is still a file to update in the sdk to make it work.